charts/jx-git-operator/values.yaml

```diff
@@ -22,8 +22,22 @@ image:
   tag: "latest"
   pullPolicy: IfNotPresent
 
+gitops-image:
+  repository: ghcr.io/jenkins-x/jx-boot
```

Comment:
Do we need jx boot or can we just use the gitops image?

Reply:
jx boot image used for gcpods job, this image contain jx and jx gitops working fine :)